library/src/enabled/java/com/telefonica/tweaks/Tweaks.kt

```diff
@@ -139,7 +139,7 @@ fun NavGraphBuilder.addTweakGraph(
             }
         }
 
-        tweaksGraph.categories.forEach { category ->
+        tweaksGraph.categories.iterator().forEach { category ->
```

Comment:
I want to take a look at this, because I was getting this error after updating the target sdk:
https://stackoverflow.com/questions/44751469/kotlin-extension-functions-suddenly-require-api-level-24

but I guess (and hope) this is not the best solution

Reply:
I had the same error when trying to upgrade AGP (I didn't find a solution)

Reply:
I wanted to check that to see myself what is happening but... nothing by my IDE is sending a warn or error.
What is suppose to happen?

![Screenshot 2022-12-20 at 16 20 22](https://user-images.githubusercontent.com/8106237/208701810-a2c66263-89c1-4dfb-8b9b-2b5c1fab980f.png)

Reply:
it fails when trying to compile